### PR TITLE
Describe included maps with examples.

### PR DESCRIPTION
**Pull request by gkellogg**
For #19.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/json-ld-syntax/pull/207.html" title="Last updated on Jul 25, 2019, 11:36 PM UTC (69f234d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/json-ld-syntax/207/7fc9975...69f234d.html" title="Last updated on Jul 25, 2019, 11:36 PM UTC (69f234d)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12246,6 +12515,13 @@ <h2>Keywords</h2>
       This keyword is described further in <a class="sectionRef" href="#context-definitions"></a>,
       and <a class="sectionRef" href="#default-vocabulary"></a>.
     </dd>
+    <dt><code>@included</code></dt><dd>
+      The `@included` keyword MUST NOT be aliased and MAY be used
```
Comment (azaroth42) [inline index.html]: In the examples it's aliased to `included` ?

**Comment by gkellogg** [comment]
> Typo: consolodated for consolidated ... otherwise great :)

Huh? I don't see "consolodated" anywhere in the spec.

File: index.html [hunk 2]
```diff
@@ -8270,6 +8280,243 @@ <h3>Using the Document Base for the Default Vocabulary</h3>
   <p class="note"><a>Type maps</a> are a new feature in JSON-LD 1.1, requiring
     <a>processing mode</a> set to <code>json-ld-1.1</code>.</p>
 </section>
+
+<section class="informative changed"><h3>Shared Value Indexing</h3>
+  <p>It is sometimes the case that a nested data structure will repeat information.
+    Using a <a>node reference</a> provides some opportunities for consolodating
```
Comment (azaroth42) [inline index.html]: Sorry, "consolodating" here.

**Comment by azaroth42** [comment]
Close in favor of #208?

**Comment by gkellogg** [comment]
Yes, we should close the others once there’s consensus on the included blocks treatment. Practically speaking, that’s probably after the meeting on Friday.

**Comment by gkellogg** [comment]
Closed in favor of #208.

**Pull request closed without merging**